uwp/Microsoft.Data.Sqlite.UWP/Microsoft.Data.Sqlite.UWP.csproj

```diff
@@ -0,0 +1,188 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{D1E1F777-AF5F-4EF2-9100-6156D4D5FF1D}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.Data.Sqlite</RootNamespace>
+    <AssemblyName>Microsoft.Data.Sqlite</AssemblyName>
```

Comment:
Maybe I'm doing something wrong, but I could not get the UWP test project to reference the DNX class library. I had to add this as a Universal class library for the UWP testing.